libs/designer/src/lib/ui/panel/panelTabs/parametersTab/index.tsx

```diff
@@ -128,7 +128,7 @@ const ParameterSection = ({
   expressionGroup: TokenGroup[];
 }) => {
   const dispatch = useDispatch<AppDispatch>();
-  const [sectionExpanded, setSectionExpanded] = useState<boolean>(false);
+  const [sectionExpanded, setSectionExpanded] = useState<boolean>(true);
```

Comment:
Any reason to change this?

Reply:
We never have it expanded in our GA, if they want it can be done in a separate PR.

Reply:
I think expanding it is fine if we're not defaulting it(Last I checked we weren't)

Agreed though it should be a separate PR